src/modules/fetcher/FetcherModule.js

```diff
@@ -10,19 +10,20 @@ export type FetchResultType = { [path: string]: {| lastUpdate: string, url: stri
 type ProgressCallbackType = (progress: number) => void
 
 class FetcherModule {
-  static currentlyFetching = false
+  // TODO NATIVE-264: Correctly handle already fetching
+  // static currentlyFetching = false
 
   async fetchAsync (
     targetFilePaths: TargetFilePathsType,
     progress: ProgressCallbackType
   ): Promise<FetchResultType> {
-    if (FetcherModule.currentlyFetching) {
-      return Promise.reject(new Error('Already fetching!'))
-    }
+    // if (FetcherModule.currentlyFetching) {
```

Comment:
Test whether this works on iOS and Android.

If this works on both platforms this would make the task NATIVE-264 easier. ALso note it there.

Reply:
I don't have a possibility to test this on iOS, but this is only temporary and should be fixed in NATIVE-264, so I don't see exactly why this should be done here.

Reply:
Hm I'm not sure whether we should trade one bug for an other one. I think this should be easily testable on browserstack, right?

Else I have a spare iphone for such tasks now around.

Reply:
So I don't really understand your point. By doing this, we don't trade a bug for another one since the `already fetching` bug is probably already existing and if not, this isn't a bug at all. Apart from that, there is already an issue to check if the problem still exists and fix it if so.
If you like though, I could also revert the `FetcherModule`, so everything stays the same.

Reply:
My point is that afaik removing the `throw` does not change anything. It just adds a very weird crash. If I remember correctly the app just crashes without any error.

I'm not really sure what the error is in this task because it is not documented in the issue. If the error is "Already fetching!" then the changes above do not fix it. If there is an another error shown then the above is probably a fix.

So if the latter is true then just revert the changes in FetcherModule and we will address this in `NATIVE-264`.

Reply:
Ah yeah ok now I get it. Yeah I'll just revert the FetcherModule as it doesn't make sense to do it in this task.